releases/v0.23.md

```diff
@@ -0,0 +1,266 @@
+# v0.23 🚀
+
+This document contains all release notes pertaining to the `v0.23.x` releases of
+Vela.
+
+## For Vela Administrators
+
+- When upgrading from `v0.23`, please take note of the [migration information](/migrations/v0.23/deploy-recovery.md)
+
+## v0.23.0
+
+### 📣 Highlights
+
+
+### 💥 Breaking Changes
+
+#### Admins:
+- `v0.23.x` introduces a new `deployments` table that will be created automatically by default. However, if you are running the `server` component with the `VELA_DATABASE_SKIP_CREATION` set to `true`, you will need to manually create this table.
+
+  - https://github.com/go-vela/server/pull/1031
+
```

Comment:
We should add a Users section that talks about the change to external contributor PRs (forks). That newly created repos will default to require someone with write permissions to allow PRs to run.

Reply:
Added